pkgs/by-name/tu/turtle/package.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, fetchFromGitLab
+, cacert
+, gobject-introspection
+, gtk4
+, glib
+, libadwaita
+, meld
+, python3
+, wrapGAppsHook4
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "turtle";
+  version = "0.6";
+  src = fetchFromGitLab {
```

Comment:
A new line here and there similar to other packages would help readability a lot.